projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -4,6 +4,9 @@ import { addQueryArgs } from '@wordpress/url';
 import '../css/jetpack-admin-jitm.scss';
 
 jQuery( document ).ready( function ( $ ) {
+	// Site ID will be automatically added to the request.
+	const JITM_ENDPOINT_URL = `/wpcom/v2/jitm-v2`;
```

Comment:
This change will require 170707-ghe-Automattic/wpcom

Reply:
Sorry this is late feedback - why is the endpoint called "jitm-v2" - the URL already has v2 in it...